index.html

```diff
@@ -51,123 +51,124 @@
-  <script src="assets/bootstrap/js/bootstrap.min.js"></script>
-  <script src="assets/smooth-scroll/SmoothScroll.js"></script>
-  <script src="assets/dropdown/js/script.min.js"></script>
-  <script src="assets/touchSwipe/jquery.touchSwipe.min.js"></script>
-  <script src="assets/viewportChecker/jquery.viewportchecker.js"></script>
-  <script src="assets/jarallax/jarallax.js"></script>
-  <script src="assets/theme/js/script.js"></script>
-  
-  
-  <input name="animation" type="hidden">
-   <div id="scrollToTop" class="scrollToTop mbr-arrow-up"><a style="text-align: center;"><i class="mbr-arrow-up-icon"></i></a></div>
-  </body>
-</html>
\ No newline at end of file
+
+<!-- /Google Analytics -->
+
+
+<section id="menu-0">
+
+    <nav class="navbar navbar-dropdown bg-color transparent navbar-fixed-top">
+        <div class="container">
+
+            <div class="mbr-table">
+                <div class="mbr-table-cell">
+
+                    <div class="navbar-brand">
+                        <a href="index.html" class="navbar-logo"><img src="assets/images/drawing-lowest-no-cropped-146x128.png" alt="Mobirise"></a>
+                        <a class="navbar-caption" href="index.html">DOCUMENTATION</a>
+                    </div>
+
+                </div>
+                <div class="mbr-table-cell">
+
+                    <button class="navbar-toggler pull-xs-right hidden-md-up" type="button" data-toggle="collapse" data-target="#exCollapsingNavbar">
+                        <div class="hamburger-icon"></div>
+                    </button>
+
+                    <ul class="nav-dropdown collapse pull-xs-right nav navbar-nav navbar-toggleable-sm" id="exCollapsingNavbar"><li class="nav-item dropdown open"><a class="nav-link link dropdown-toggle" data-toggle="dropdown-submenu" href="https://mobirise.com/" aria-expanded="true">ADD-ONS</a><div class="dropdown-menu"><a class="dropdown-item" href="enrich.html">Enrich<br></a><a class="dropdown-item" href="kaleidoscope.html">Kaleidoscope</a></div></li><li class="nav-item nav-btn"><a class="nav-link btn btn-white btn-white-outline" href="http://www.blenderskool.cf" target="_blank">GO TO THE MAIN SITE</a></li></ul>
+                    <button hidden="" class="navbar-toggler navbar-close" type="button" data-toggle="collapse" data-target="#exCollapsingNavbar">
+                        <div class="close-icon"></div>
+                    </button>
+
+                </div>
+            </div>
+
+        </div>
+    </nav>
+
+</section>
+
+<section class="engine"><a rel="external" href="https://mobirise.com">Mobirise</a></section><section class="mbr-section mbr-section-hero mbr-section-full mbr-parallax-background mbr-after-navbar" id="header1-1" style="background-image: url(assets/images/background-2000x1453.jpg);">
+
+    <div class="mbr-overlay" style="opacity: 0.2; background-color: rgb(0, 0, 0);"></div>
+    <div class="mbr-overlay" style="opacity: 0.2; background-color: rgb(40, 80, 90);"></div>

```

Comment:
Background Image without any more overlay colors looks much better. Making the background more dull.